uint32 agent_role
uint32 uID
nav_msgs/Odometry odom
uint32 degree
uint32 isAP
